app/Models/Country.php

```diff
@@ -10,4 +10,11 @@ class Country extends Model
     use HasFactory;
 
     protected $fillable = ['name', 'short_code'];
+
+    protected $hidden = ['created_at', 'updated_at'];
+
+    public function achievements()
```

Comment:
@jstarnate I don't really approve the mix of namings here.

- results
- achievements
- winners

I will shoot a separate video explaining my take on this, but generally it should be a pivot table like `country_sport` and not `results`

Reply:
@PovilasKorop This might sound ridiculous but since the challenge is about Olympics results, I decided to give them those names. I thought the table name `country_sport` would sound like "popular sports in a particular country" or "sports invented in a particular country". I will practice this convention in my future Laravel projects, thanks! :)